kfs-core/src/main/resources/edu/arizona/kfs/gl/businessobject/datadictionary/DocumentTypeLiteBo.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+    <bean id="DocumentTypeLiteBo" parent="BusinessObjectEntry">
+        <property name="businessObjectClass" value="edu.arizona.kfs.gl.businessobject.lite.DocumentTypeLiteBo" />
+        <property name="objectLabel" value="DocumentTypeLiteBo" />
+        <property name="attributes">
+            <list>
+                <ref bean="DocumentTypeLiteBo-documentTypeId" />
+                <ref bean="DocumentTypeLiteBo-docTypeParentId" />
+                <ref bean="DocumentTypeLiteBo-name" />
+                <ref bean="DocumentTypeLiteBo-version" />
+                <ref bean="DocumentTypeLiteBo-active" />
+                <ref bean="DocumentTypeLiteBo-currentInd" />
+                <ref bean="DocumentTypeLiteBo-label" />
+                <ref bean="DocumentTypeLiteBo-previousVersionId" />
+                <ref bean="DocumentTypeLiteBo-description" />
+                <ref bean="DocumentTypeLiteBo-unresolvedDocHandlerUrl" />
+                <ref bean="DocumentTypeLiteBo-postProcessorName" />
+                <ref bean="DocumentTypeLiteBo-blanketApprovePolicy" />
+                <ref bean="DocumentTypeLiteBo-routingVersion" />
+                <ref bean="DocumentTypeLiteBo-actualNotificationFromAddress" />
+                <ref bean="DocumentTypeLiteBo-actualApplicationId" />
+                <ref bean="DocumentTypeLiteBo-customEmailStylesheet" />
+                <ref bean="DocumentTypeLiteBo-documentTypeSecurityXml" />
+                <ref bean="DocumentTypeLiteBo-versionNumber" />
+                <ref bean="DocumentTypeLiteBo-blanketApproveWorkgroupId" />
+                <ref bean="DocumentTypeLiteBo-reportingWorkgroupId" />
+                <ref bean="DocumentTypeLiteBo-workgroupId" />
+                <ref bean="DocumentTypeLiteBo-unresolvedHelpDefinitionUrl" />
+                <ref bean="DocumentTypeLiteBo-objectId" />
+                <ref bean="DocumentTypeLiteBo-unresolvedDocSearchHelpUrl" />
+                <ref bean="DocumentTypeLiteBo-documentId" />
+                <ref bean="DocumentTypeLiteBo-authorizer" />
+            </list>
+        </property>
+    </bean>
+
+<!--     Attribute Definitions -->
+    <bean id="DocumentTypeLiteBo-documentTypeId"                parent="DocumentType-documentTypeId"                                                            />

```

Comment:
Formatting again -- if you could find a way to keep the columns *while* pulling in the last angle bracket, I could compromise.